include/maliput_sparse/geometry/utility/geometry.h

```diff
@@ -38,23 +38,33 @@ namespace geometry {
 namespace utility {
 
 /// Holds the result of #GetBoundPointsAtP method.
+/// @tparam T The coordinate type.
+template <typename T>
```

Comment:
I have a slight preference for CoordinateT instead of T. It is self-documented what it is expected to be.

Reply:
Done.